config/rbac/role.yaml

```diff
@@ -6,6 +6,13 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
```

Comment:
This is required at the cluster level as we need access across cluster to generate events for violating resources in their own namespaces from audit results.

Reply:
Are we tying violations in audit to the violating resource or to the gatekeeper audit deployment?

I thought we were tying everything to the deployment so we would have a place to stick violations if we ever have namespace-scoped constraints?